### ISSUE TITLE
Improve init command

### ISSUE THREAD

**Issue by cornerman** [protontypes/openselery#63]
Resulting yaml should have comments and proper ordering of keys

**Comment by Ly0n** [comment]
The current init command does only ask for the Bitcoin Address. We should create selery.yml configuration editor that checks an visualizes input and avoids missconfigurations.
* Check if Bitcoin address is valid
* Display entered bitcoin amount in € or $
* .....

**Comment by Ly0n** [comment]
We could also print the markdown badge code for the project README that is gone be configured so that the user can copy / past it.

**Comment by cornerman** [comment]
We also need a `openselery check` command to check for sanity and provide info like how much money configuired btc is worth.

**Comment by krux02** [comment]
I think about the setting "simulation". Running with ``simulation = true`` and then go into the settings file, change simulation to ``false`` and then run it again isn't a good workflow. I think something like ``selery check`` or ``selery status`` should replace the ``selery run`` with the ``simulation = true`` setting.

**Comment by cornerman** [comment]
I agree with `Simulation = true` in the config. I also think it should be controlled over the commandline.

**Comment by cornerman** [comment]
We could also add an option like `--payout` to the `run` command to enfore people to actively say they want payout. So there is no accidental transactions.

**Comment by Ly0n** [comment]
@krux02 : This example works for me to get the Bitcoin price via an open API.
https://gist.github.com/mattvukas/9146611

**Comment by Ly0n** [comment]
In the past I created cool CLI with this tool:
https://github.com/prompt-toolkit/python-prompt-toolkit
Maybe it is useful in this use case or maybe not.

**Comment by Ly0n** [comment]
Relates to #131 and #96

**Comment by Ly0n** [comment]
also relates to #63
